### DISCUSSION TITLE
Document updates in live editing mode

### DISCUSSION THREAD

**Discussion by terravi** [ONLYOFFICE/DocumentServer#1758] [Q&A]
Guest users (read-only view mode) can't see updates in live editing. Are we making some mistake or is this feature not provided by the product?

**Comment by ShockwaveNN** [comment]
I think it's a duplicate of https://github.com/ONLYOFFICE/DocumentServer/issues/552

Right now we're working on view mode with real-times update, but I cannot provide any time frame for it's release
